README.md

```diff
@@ -5,7 +5,7 @@
 Ignition Fuel Tools is composed by a client library and command line tools for
 interacting with Ignition Fuel servers.
 
-  [http://bitbucket.org/ignitionrobotics/ign-fuel-tools](http://bitbucket.org/ignitionrobotics/ign-fuel-tools)
+  [http://github.com/ignitionrobotics/ign-fuel-tools](http://github.org/ignitionrobotics/ign-fuel-tools)

```

Comment:
Not sure why this ended up as `github.org` rather than `.com`

Reply:
Yeah weird, I'll keep an eye on those